<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamaraView</class>
 <widget class="QDialog" name="CamaraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>45</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnReturn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMsg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="display_mode_1">
      <property name="text">
       <string>模式1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="display_mode_2">
      <property name="text">
       <string>模式2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="display_mode_3">
      <property name="text">
       <string>模式3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="display_mode_4">
      <property name="text">
       <string>模式4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="display_mode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示模式</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QCheckBox" name="osd_add_date">
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="osd_add_timer">
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="osd_add_ch">
      <property name="text">
       <string>通道号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="osd_add_resolution">
      <property name="text">
       <string>分辨率</string>
      </property>
      <property name="iconSize">
       <size>
        <width>29</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="osd_add_set">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OSD设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>315</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QCheckBox" name="display_ch_1">
      <property name="text">
       <string>CH1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="display_ch_2">
      <property name="text">
       <string>CH2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="display_ch_3">
      <property name="text">
       <string>CH3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="display_ch_4">
      <property name="text">
       <string>CH4</string>
      </property>
      <property name="iconSize">
       <size>
        <width>29</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="display_channel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示通道</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
